### PR TITLE
{math}[GCCcore/12.3.0] SpectrA v1.0.1

### PR DESCRIPTION
**Pull request by benczaja**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/SpectrA/SpectrA-1.0.1-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+name = 'SpectrA'
+version = '1.0.1'
+
+homepage = 'https://spectralib.org/'
+description = """Spectra stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++
+ library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/yixuan/spectra/archive/refs/tags']
```
[the remark this answers is not in the document]
Comment (benczaja) [reply easybuild/easyconfigs/s/SpectrA/SpectrA-1.0.1-GCCcore-12.3.0.eb]: Change incorperated

**Comment by akesandgren** [comment]
@boegelbot Please test @ jsc-zen3

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 545.29.06, Python 3.10.12
See https://gist.github.com/akesandgren/3a8e7c43c5dd288421018ae5263a81ef for a full test report.

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @benczaja!

**Pull request merged**